wikipron/config.py

```diff
@@ -120,13 +124,20 @@ def _get_casefold(self, casefold: bool) -> Callable[[Word], Word]:
         return str.casefold if casefold else lambda word: word  # noqa: E731
 
     def _get_process_pron(
-        self, no_stress: bool, no_syllable_boundaries: bool, no_segment: bool
+        self,
+        no_stress: bool,
+        no_syllable_boundaries: bool,
+        no_segment: bool,
+        no_tone: bool,
     ) -> Callable[[Pron], Pron]:
         processors = []
         if no_stress:
             processors.append(functools.partial(re.sub, r"[ˈˌ]", ""))
         if no_syllable_boundaries:
             processors.append(functools.partial(re.sub, r"\.", ""))
+        if no_tone:
+            processors.append(functools.partial(unicodedata.normalize, "NFD"))
```

Comment:
This line 139 is no longer necessary given what you've done. It ought to be no-op.